### PR TITLE
[16.0] [MIG] stock location position

### PR DESCRIPTION
**Pull request by TDu**

### CODE REVIEW

File: stock_location_position/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    "name": "Stock Location Position",
+    "version": "16.0.1.0.0",
+    "author": "BCIM, Okia, Camptocamp, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/stock-logistics-warehouse",
+    "summary": "Add coordinate attributes on stock location.",
+    "category": "Stock Management",
+    "depends": ["stock"],
+    "data": ["views/stock_location.xml"],
+    "installable": True,
+    "development_status": "Alpha",
```
Comment (rousseldenis) [inline stock_location_position/__manifest__.py]: @TDu Still Alpha ?

Comment (TDu) [reply stock_location_position/__manifest__.py]: Good remark, I changed it to stable.

**Review by rousseldenis — APPROVED** [review]
Code review

File: stock_location_position/README.rst [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+* Row
+* Rack
+* Level
+
+and renames position (XYZ) to box (XYZ).
+
+.. IMPORTANT::
+   This is an alpha version, the data model and design can change at any time without warning.
+   Only for development or testing purpose, do not use in production.
+   `More details on development status <https://odoo-community.org/page/development-status>`_
```
Comment (BT-cjimeno) [inline stock_location_position/README.rst:40]: Is this still needed when using "Production/Stable" in the manifest?
There are also some links pointing to 14.0, I have no idea if those need to be changed or some OCA git bot will take care of those, I see no reference to the README here:
https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-16.0

**Review by BT-cjimeno — APPROVED** [review]
Code review

**Comment by lmignon** [comment]
@rousseldenis Can you reopen this one and merge it...

File: stock_location_position/README.rst (hunk 2 above)
Comment (rousseldenis) [reply stock_location_position/README.rst:40]: @BT-cjimeno FYI, README.rst will be regenerated at merge if fragments exist in readme/ directory (contributor can do it if he wants to with oca-gen-addon-readme command, but not mandatory)

Comment (BT-cjimeno) [reply stock_location_position/README.rst:40]: Thanks for the info :)

**Pull request merged**